system/mp/mpcomm.cpp

```diff
@@ -2160,7 +2175,7 @@ int CMPConnectThread::run()
                 traceSlowReadTms("MP: initial accept packet from", sock, &connectHdr, sizeof(connectHdr.id), sizeof(connectHdr), rd, CONFIRM_TIMEOUT, CONFIRM_TIMEOUT_INTERVAL);
                 if (0 == rd)
                 {
-                    if (parent->mpTraceLevel > 1)
+                    if (parent->mpTraceLevel > (ExtraneousMsgThreshold + 10))
```

Comment:
until the default logging level is lowered, this is still below the default.
I would introduce a constant somewhere like MyVerboseLoggingThreadhold=120 for now
can switch to ExtraneousMsgThreshold  when default level changed.